packages/blade/src/components/BottomSheet/_decisions/decisions.md

```diff
@@ -0,0 +1,225 @@
+# Bottom Sheet
+
+Bottom sheet is a container that presents additional information to a user in an overlay.
+
+This document outlines the API of `BottomSheet` component.
+
+<img src="./bottom-sheet-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - BottomSheet](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=26477%3A578228&t=1RMqjs99P33Udj0C-0)
+
+## API
+
+Sample usage:
+
+```jsx
+import { BottomSheet } from '@razorpay/blade';
+
+<BottomSheet open={boolean} snapPoints={[]}>
+  <BottomSheetHeader>
+    <BottomSheetHeaderLeading title="Payments Links" prefix={BladeIcon} />
+    <BottomSheetHeaderTrailing visual={Link | Action | Icon} />
+  </BottomSheetHeader>
```

Comment:
Can we make this similar to ActionListHeader?
```jsx
<BottomSheetHeader 
  title="Payments Links" 
  leading={<BottomSheetHeaderIcon icon={BladeIcon}} 
  trailing={<BottomSheetHeaderIcon icon={Icon} /> | <BottomSheetHeaderLink /> | <BottomSheetHeaderButton />}
/>
```

Reply:
Wanted to keep the API similar to Card since the BottomSheet is basically a "fancy" modal. and the anatomy of it is very similar, eg: the footer/footerActions are 1:1 with the API & functionality with Card.

But we can discuss this with kamlesh once.

Reply:
Why is ActionListHeader and CardHeader different in terms of leading and trailing? Do we have a strong rationale?

Reply:
as far I remember we went with that API is CardHeader because even inside leading, it had prefix and suffix of it's own. This part -
<img width="575" alt="image" src="https://user-images.githubusercontent.com/30949385/222336704-def5c81b-48b9-4be1-b5bd-b4d5a841df19.png">

So we had to go with `leading` as a prop, then there wasn't any way to introduce this prefix and suffix. right @anuraghazra 🤔 ?

BottomSheetHeader doesn't have that usecase no?

Reply:
https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Dropdown/_decisions/decisions.md#components-as-jsx-props-instead-of-compound-components this was the reason to go with props in ActionList